_partials/_prometheus-integrate.md

```diff
@@ -2,119 +2,224 @@ import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.md
+         ```shell
+         export DATA_SOURCE_NAME="postgres://monitoring:<password>@<host>.tsdb.cloud.timescale.com:<port>/tsdb?sslmode=require"
+         ./postgres_exporter
+         ```
+       - Docker:
+         ```shell
+         docker run -d \ 
+            -e DATA_SOURCE_NAME="postgres://monitoring:<password>@<host>.tsdb.cloud.timescale.com:<port>/tsdb?sslmode=require" \ 
+            -p 9187:9187 \ 
+            prometheuscommunity/postgres-exporter
+         ```
+
+    1. Configure the following labels for your $SERVICE_SHORT in PostgreSQL Exporter:
+
+       - `project_id`
+       - `service_id`
+       - `region`
+
+    1. Check the metrics for your $SERVICE_SHORT in the Prometheus format:
 
-   1. In your Prometheus installation, update `prometheus.yml` to point to the exporter URL as a scrape target:
+        - Browser:
 
-      ```yml
-      scrape_configs:
-       - job_name: "timescaledb-exporter"
-         scheme: https
+          Navigate to `http://<exporter-host>:9187/metrics`.
+
+        - Command line:
+           ```shell
+           curl http://<exporter-host>:9187/metrics
+           ```        
+
+1. **Configure Prometheus to scrape metrics**
+
+    1. In your Prometheus installation, update `prometheus.yml` to point to your PostgreSQL Exporter instance as a scrape
+       target. In the following example, you replace `<exporter-host>` with the hostname or IP address of the PostgreSQL
+       Exporter.
+
+       ```yaml
+       global:
+         scrape_interval: 15s
+       
+       scrape_configs:
+       - job_name: 'postgresql'
          static_configs:
-           - targets: ["my-exporter-url"]
-         basic_auth:
-           username: "user"
-           password: "pass"
-      ```
-   
-      See the [Prometheus documentation][scrape-targets] for details on configuring scrape targets. 
-
-      You can now monitor your $SERVICE_SHORT metrics. Use the following metrics to check the service is running correctly:
-
-      *   `timescale.cloud.system.cpu.usage.millicores`
-      *   `timescale.cloud.system.cpu.total.millicores`
-      *   `timescale.cloud.system.memory.usage.bytes`
-      *   `timescale.cloud.system.memory.total.bytes`
-      *   `timescale.cloud.system.disk.usage.bytes`
-      *   `timescale.cloud.system.disk.total.bytes`
-
-      Additionally, use the following tags to filter your results.
-
-      |Tag|Example variable| Description                |
-      |-|-|----------------------------|
-      |`host`|`us-east-1.timescale.cloud`|                            |
-      |`project-id`||                            |
-      |`service-id`||                            |
-      |`region`|`us-east-1`| AWS region                 |
-      |`role`|`replica` or `primary`| For $SERVICE_SHORT with replicas |
-   
+          - targets: ['<exporter-host>:9187'] 
+       ```
+
+       If `prometheus.yml` has not been created during installation, create it manually. If you are using Docker, you can
```

Comment:
"If `prometheus.yml` has not been created during installation, create it manually." seems pointless as you have just told them to do it

Reply:
We're telling to **update** the file, but there is a possibility it hasn't been created.